lib/msal-browser/src/controllers/IController.ts

```diff
@@ -25,9 +25,10 @@ import { EventHandler } from "../event/EventHandler";
 import { AuthenticationResult } from "../response/AuthenticationResult";
 import { EventCallbackFunction } from "../event/EventMessage";
 import { ClearCacheRequest } from "../request/ClearCacheRequest";
+import { InitializeApplicationRequest } from "../request/InitializeApplicationRequest";
 
 export interface IController {
-    initialize(): Promise<void>;
+    initialize(request?: InitializeApplicationRequest): Promise<void>;
```

Comment:
V4 plan to make this required?  Can we flag this as a code warning / open a bug to track?

Reply:
I don't think we have plans to make it mandatory at the moment, though I can add a TO-DO here

Reply:
Can we add it to the internal ADO?